lib/sles4sap/console_redirection.pm

```diff
@@ -7,10 +7,12 @@
 package sles4sap::console_redirection;
 use strict;
 use warnings;
+use testapi qw(is_serial_terminal :DEFAULT);
```

Comment:
`is_serial_terminal` does not seem to be exported same as other functions. 
To be honest I just copy and pasted this from other test which used it.